src/classes/PurchaseOrderControllerTest.cls

```diff
@@ -0,0 +1,93 @@
+@isTest
+private class PurchaseOrderControllerTest {
+
+	static PurchaseOrderController ctrl = new PurchaseOrderController(
+            new ApexPages.StandardController(new PurchaseOrder__c()));
+    
+    @isTest static void testCancel()
```

Comment:
method name should be in following format:
methodName_shortDescription_finalResult